charts/virtual-kubelet/templates/deployment.yaml

```diff
@@ -20,12 +20,20 @@ spec:
           value: {{ .Values.env.aciResourceGroup }}
         - name: ACI_REGION
           value: {{ default "westus" .Values.env.aciRegion }}
+        - name: APISERVER_CERT
+          value: {{ .Values.env.apiserverCert | quote }}
+        - name: APISERVER_KEY
+          value: {{ .Values.env.apiserverKey | quote }}
```

Comment:
As the apiserver key is a secret, can we mount the apiserver key and cert as secret volume?